# Lab | Inferential statistics

Instructions
1. It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?
    * Set up the hypothesis test.
    * Write down all the steps followed for setting up the test.
    * Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.


2. If you finished the previous question, please go through the code for principal_component_analysis_example provided in the files_for_lab folder .



* H0 : The sample average for blood pressure is significantly different than the population avegrave for blood pressure

    * We assume that the standardard deviation is the same for the sample than the population

In [1]:
import math as m
import pandas as pd

In [2]:
μ = 120
n = 100
sysBP = 130.1
std = 21.21
alpha = 1.96

In [3]:
z = (sysBP - μ) / (std/m.sqrt(n))
z

4.761904761904759

* The z score for our sample data 4.76 is bigger than the alpha 1.96, therefore we reject the null hypothesis.